react_ws_src/src/helpers/computer_moves.js

```diff
@@ -0,0 +1,168 @@
+const BOARD_SIZE = 9;
```

Comment:
Although I haven't changed the board size, I remember you suggesting that could be one of the tasks I do. So although I'm not passing it into the `get_computer_move` function, or getting it from `config`, I have _parametized_ it so that it would be easier to do later.